shuttle/schedules/__init__.py

```diff
@@ -1,21 +1,25 @@
-from flask import render_template
+from flask import render_template, request
 from flask_classy import FlaskView, route
+from shuttle.db import db_functions as db
+from shuttle.schedules.google_sheets_controller import SheetsController
 
 from shuttle.shuttle_controller import ShuttleController
 
 
 class SchedulesView(FlaskView):
     def __init__(self):
         self.sc = ShuttleController()
+        self.shc = SheetsController()
 
     @route('/shuttle-stats')
     def stats(self):
         return render_template('/schedules/shuttle_stats.html')
 
     @route('/request-shuttle')
     def request(self):
+        location_list = SheetsController.grab_locations(self)
         self.sc.check_roles_and_route(['Administrator', 'Driver', 'User'])
-        return render_template('/schedules/request_shuttle.html')
+        return render_template('/schedules/request_shuttle.html', locations=location_list)
```

Comment:
You could change your variable above `location_list` to just `locations` and then instead of having `locations=location_list` you can just past `**locals` which grabs all the variables you have in the method.